applications/nrf_desktop/src/modules/Kconfig.dfu_mcumgr

```diff
@@ -20,6 +20,7 @@ menuconfig DESKTOP_DFU_MCUMGR_ENABLE
 	select FLASH_MAP
 	select ZCBOR
 	depends on BOOTLOADER_MCUBOOT
+	depends on !MCUBOOT_BOOTLOADER_MODE_RAM_LOAD
```

Comment:
Can we comment in Kconfig help that it's just a limitation of application module's implementation?